resources/views/welcome.blade.php

```diff
@@ -12,6 +12,20 @@
             <li class="list-group-item"><a href="/drinks">Drycker</a></li>
             <li class="list-group-item"><a href="/history">Historia</a></li>
         </ul>
+
+        <?php
+            $brewKey = "0cecc2de820dc46cde46804f597e1614";
+            $url = "http://api.brewerydb.com/v2/beers/?hasLabels=Y&key=$brewKey";
+            $databrew = file_get_contents($url);
+            $brew = json_decode($databrew);
+            $brewNum = rand(0, 45);
+            $brewPresent = ($brew->data[$brewNum]->name);
+            $brewPic = ($brew->data[$brewNum]->labels->icon);
```

Comment:
How do you feel about a bigger label picture?
It's possible to use 'labels->medium' or 'labels->large'

My vote is on medium =)

Reply:
se

> How do you feel about a bigger label picture?
> 
> It's possible to use 'labels->medium' or 'labels->large'
> 
> My vote is on medium =)

Second that! Tried it out and it was much easier to see the label

Reply:
Yes, much better.